bin/build.sh

```diff
@@ -5,8 +5,9 @@ mkdir -p ./release
 # let gulp build the assets
 docker run --rm --label=gulp \
     --volume=$(pwd):/srv \
+    --entrypoint /bin/bash \
     huli/gulp \
-    build --production
+    -c "gulp build --production && chown $(id -u):$(id -g) . -R"
```

Comment:
I usually just run the container as the user with `--user "$(id -u)"`. Just for extra info.

Reply:
Yes, the `-u` flag I was referring to in https://github.com/mozilla/irlpodcast/pull/174#issuecomment-463844594 is the shorthand for `--user`, and I got an error due to issues with the way the `huli/gulp` image tries to write files to the root of the container.

Reply:
Ah! I totally missed your other comment mentioning the `-u`. Makes sense. Thanks for the clarification.

Reply:
Specifically, the error I got was:
```
$ bin/build.sh
npm WARN IRLPodcast@2.0.0 license should be a valid SPDX license expression
npm ERR! Linux 4.9.125-linuxkit
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "--save-dev"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.3
npm ERR! path /.npm
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall mkdir

npm ERR! Error: EACCES: permission denied, mkdir '/.npm'
npm ERR!     at Error (native)
npm ERR!  { Error: EACCES: permission denied, mkdir '/.npm'
npm ERR!     at Error (native) errno: -13, code: 'EACCES', syscall: 'mkdir', path: '/.npm' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /srv/npm-debug.log
Error: Unable to open Config file.
 (open config-.toml: no such file or directory)
```